modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -263,38 +143,6 @@ def _width_extraction_fn(cls):
         """
         return width_fn_pandas
 
-    def length(self):
-        """
-        Get the length of the object wrapped by this partition.
-
-        Returns
-        -------
-        int
-            The length of the object.
-        """
-        if self._length_cache is None:
-            cls = type(self)
-            func = cls._length_extraction_fn()
-            preprocessed_func = cls.preprocess_func(func)
-            self._length_cache = self.apply(preprocessed_func)
-        return self._length_cache
-
-    def width(self):
-        """
-        Get the width of the object wrapped by the partition.
-
-        Returns
-        -------
-        int
-            The width of the object.
-        """
-        if self._width_cache is None:
-            cls = type(self)
-            func = cls._width_extraction_fn()
-            preprocessed_func = cls.preprocess_func(func)
-            self._width_cache = self.apply(preprocessed_func)
```

Comment:
In the current implementation, `self._width_cache` variable stores an object of partition type. 

Modin's code works because each child class gets rid of this error by overriding the method.